.jenkins/common.groovy

```diff
@@ -63,7 +62,7 @@ def runCompileCommand(platform, project, jobName, boolean debug=false)
             pushd build
 
             export PATH=/opt/rocm/bin:\$PATH
-            cmake -DCMAKE_BUILD_TYPE=${buildType} -DCMAKE_CXX_COMPILER=${compiler} -DTensile_CPU_THREADS=${buildThreads} -DTensile_ROOT=\$(pwd)/../Tensile ../HostLibraryTests
```

Comment:
This may have been of use for unrelated OOM kills

Reply:
Yea, looks like this was probably needed for gfx11 on CI. Posted PR 1849 to add this back in, but limited to gfx11 builds.